eth/api_tracer.go

```diff
@@ -915,7 +915,7 @@ func (api *PrivateDebugAPI) traceTx(ctx context.Context, message core.Message, v
 
 	// Run the transaction with tracing enabled.
 	vmenv := vm.NewEVM(vmctx, txContext, statedb, privateStateDbToUse, api.eth.blockchain.Config(), vm.Config{Debug: true, Tracer: tracer})
-	vmenv.SetCurrentTX(tx)
+	// TODO: vmenv.SetCurrentTX(tx)
```

Comment:
TODO: check this, also do trace call, and debug call test to verify it's working

Reply:
TODO: test api tracer manually